readme.md

```diff
@@ -115,7 +109,7 @@ The tests of Hoff are extensive, you may be able to get by just by running them
 when making changes to the code.  To run a specific test, use `--match` giving
 part of the test title:
 
-	$ nix shell --file default.nix -c stack test --ta '--match "part of the test title"'
+	$ nix develop --file default.nix -c cabal run spec -- --match "part of the test title"
 
 The implementation uses free monads and some of the tests replace lower level
```

Comment:
Not related to this PR, but it isn't true anymore if `effectful` has been merged.

Reply:
Ah yeah you're right. I can tack that onto this PR but there's probably more out of date docs that have to be updated.

Reply:
Yeah, I guess so. Noticed this one, so I figured I'd just write it here :eyes: